docker/proof-server-chart/values.yaml

```diff
@@ -0,0 +1,27 @@
+deployment:
+  proofServer:
+    env:
+      containerPort: "8080"
+      rustLog: "debug"
+      rustMinStack: "16777216"
+      shardSize: "4194304"
+      shardBatchSize: "0"
+      shardChunkingMultiplier: "32"
+      reconstructCommitments: "false"
+    image:
+      repository: ghcr.io/argumentcomputer/ethereum-proof-server
+      tag: latest
```

Comment:
Our tag uses latest yet we set an `appVersion` of 0.1.0, does this make sense?

Reply:
I changed `appVersion` to latest. However, as It is specified in the documentation that the user should replace the image by what they want it is likely that those will never have a 1:1 correspondence (if the user does not bother to maintain it).